### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by nater1983** [apeitheo/aletheia#8]
If you tag a release I can help with the rest of the information for SBo if you would like.

**Comment by apeitheo** [comment]
If you could help get the release up on SBo that would be very helpful. I haven't had the ability to work much at all on it the last two days or so since I've been dealing with some kind of illness, but I think within the week I can probably tag 0.9.6. If you could help get it up on SBo, that would help a lot.

As far as things left to do, in the search ('s') and view playlist (';') modes, it's still using the legacy digital voice instead of the GTTS one, but I haven't found a good working solution that doesn't slow things down significantly. I'd like to expand support for metadata tags as that was just added recently. Right now you can only search via filename, and it doesn't take the tags into consideration, for example.

Maybe all of that can wait for the next 0.9.7 release though.

I appreciate all the help so far.

**Comment by apeitheo** [comment]
I had to run the core aletheia script through shellcheck a while ago and make some similar changes to the ones I made to the SlackBuild earlier. I had forgotten to run shellcheck on the SlackBuild even though I had done the others.

Shellcheck warns if I don't quote variables and such as they could contain spaces. I'd like the shellcheck output to come back clean as it helps debug other problems that come up. It also recommended using find a little differently as it wouldn't handle filenames with the newline character. It shouldn't happen with manpages, but maybe the code will be re-used and then will handle it appropriately.

Hope all is well
